index.html

```diff
@@ -211,7 +211,7 @@
         <h1>Glossary</h1>
         <dl>
             <dt><dfn>active buttons state</dfn></dt>
-                <dd>The condition when a pointer has a non-zero value for the <code>buttons</code> property. For mouse, this is when the device has at least one button depressed. For touch, this is when there is physical contact with the digitizer. For pen, this is when the pen has physical contact with the digitizer.</dd> 
+                <dd>The condition when a pointer has a non-zero value for the <code>buttons</code> property. For mouse, this is when the device has at least one button depressed. For touch, this is when there is physical contact with the digitizer. For pen, this is when the pen either has physical contact with the digitizer, or has at least one button depressed while hovering.</dd>
```

Comment:
I don't think it needs an extra qualifier. If the device doesn't have a button, then that portion of the conditional will just never be met, but it's an or so it doesn't matter.